src/sdkg.rs

```diff
@@ -498,11 +455,11 @@ impl<N: NodeIdT, PK: PublicKey> SyncKeyGen<N, PK> {
         let ser_val = self
             .sec_key
             .decrypt(&values[our_idx as usize])
-            .map_err(|_| AckFault::DecryptValue)?;
+            .ok_or(AckFault::DecryptValue)?;
         let val = bincode::deserialize::<FieldWrap<Fr>>(&ser_val)
             .map_err(|_| AckFault::DeserializeValue)?
             .into_inner();
-        if part.commit.evaluate(our_idx + 1, sender_idx + 1) != G1Affine::one().mul(val) {
```

Comment:
I'm not knowledgable about this but is `one()` here the same as `generator()`?

Reply:
I'm not knowledgeable as well :sweat_smile: , but just followed the function call and `one()` pointed me to a macro called `cruve_impl!` 
![image](https://user-images.githubusercontent.com/73495426/177816943-47548439-1e28-41cb-9c59-dc4333ddca93.png)
and I found that `one()` returns `CurveAffine::get_generator()`. And our generator is from the trait `blsttc::group::prime::PrimeCurveAffine`, so it must be the one right?
![image](https://user-images.githubusercontent.com/73495426/177817096-9e301638-4d7f-413b-aab3-4b4add179f82.png)